bowling/src/frameParser.cpp

```diff
@@ -0,0 +1,54 @@
+#include "frameParser.hpp"
+
+#include <iostream>
+#include <iterator>
+#include <algorithm>
+
+parsedFrame FrameParser::parse(const std::string& gamePoints) {
+  std::vector<size_t> parsed {};
+  size_t last_parsed_char{};
+  auto tokenized = Tokenizer::splitToFramesAlt(gamePoints);
```

Comment:
Separate into 2 methods, f.e. one for readName, second for readSequence:
```suggestion
  auto name= readName(line);
  auto sequence = readSequence(line);
```

Reply:
Done.